core/math/math_funcs.h

```diff
@@ -215,15 +215,20 @@ class Math {
 	static _ALWAYS_INLINE_ double lerp(double p_from, double p_to, double p_weight) { return p_from + (p_to - p_from) * p_weight; }
 	static _ALWAYS_INLINE_ float lerp(float p_from, float p_to, float p_weight) { return p_from + (p_to - p_from) * p_weight; }
 
+	static _ALWAYS_INLINE_ double angle_diff(double from, double to) {
+		double diff = fposmod(to - from, (double)Math_TAU);
+		return fmod(2.0 * diff, Math_TAU) - diff;
+	}
+	static _ALWAYS_INLINE_ float angle_diff(float from, float to) {
+		float diff = fmod(to - from, (float)Math_TAU);
```

Comment:
```suggestion
	static _ALWAYS_INLINE_ double angle_diff(double p_from, double p_to) {
		double diff = fposmod(p_to - p_from, Math_TAU);
		return fmod(2.0 * diff, Math_TAU) - diff;
	}
	static _ALWAYS_INLINE_ float angle_diff(float p_from, float p_to) {
		float diff = fmod(p_to - p_from, (float)Math_TAU);
```

One of these is using `fposmod` and one is using `fmod`. I'm not sure which one is correct in this particular case, but I'm sure they need to have the same behavior.